client/src/main/java/org/dst/client/cmd/ArgsParseUtil.java

```diff
@@ -0,0 +1,57 @@
+package org.dst.client.cmd;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import org.dst.common.exception.DstException;
+
+public class ArgsParseUtil {
+
+  private static final String DEFAULT_VERSION = "0.1.0";
+
+  @Parameter(names = {"--help", "-help"}, description = "show the help command",
+          help = true, order = -1)
+  private static boolean HELP = false;
+  @Parameter(names = {"-h", "-H"}, description = "specify the host of server to connect", order = 1)
+  private static String HOST = "127.0.0.1";
+  @Parameter(names = {"-p", "-P"}, description = "specify the port of server to connect", order = 2)
+  private static String PORT = "8082";
+  @Parameter(names = {"-v", "-version", "-V"}, description = "show the version of Dst",
+      help = true, order = 3)
+  private static boolean VERSION = false;
+
+  /**
+   * @param args the args like [-h,127.0.0.1,-p,8082] or [-h,127.0.0.1] or [-p,8082]
+   * @return list://127.0.0.1:8082
+   */
+  public static String covertArgsToAddress(String[] args) throws DstException {
+
+    StringBuilder sb = new StringBuilder();
+
+    ArgsParseUtil argsParseUtil = new ArgsParseUtil();
+    JCommander jcommander = JCommander.newBuilder().addObject(argsParseUtil).build();
+    try {
+      jcommander.parse(args);
+    } catch (Exception e) {
+      jcommander.setProgramName("dst");
+      jcommander.usage(sb);
```

Comment:
What is the meaning of this assignment before throwing an exception?

Reply:
This can make cmd show:
```shell
Usage: dst [options]
  Options:
    -h, -H
      specify the host of server to connect
      Default: 127.0.0.1
    -p, -P
      specify the port of server to connect
      Default: 8082
    -v, -version, -V
      show the version of Dst
    --help, -help
      show the help command
```

Reply:
awesome！